src/Nethermind/Nethermind.DataMarketplace.Infrastructure/INdmApi.cs

```diff
@@ -0,0 +1,43 @@
+using Nethermind.Api;
+using Nethermind.Core;
+using Nethermind.DataMarketplace.Channels;
+using Nethermind.DataMarketplace.Consumers.Shared;
+using Nethermind.DataMarketplace.Core;
+using Nethermind.DataMarketplace.Core.Configs;
+using Nethermind.DataMarketplace.Core.Domain;
+using Nethermind.DataMarketplace.Core.Services;
+using Nethermind.DataMarketplace.Infrastructure.Persistence.Mongo;
+using Nethermind.Db;
+using Nethermind.Facade.Proxy;
+using Nethermind.Serialization.Rlp;
+
+namespace Nethermind.DataMarketplace.Infrastructure
+{
+    public interface INdmApi : INethermindApi
+    {
+        public IConfigManager? ConfigManager { get; set; }
```

Comment:
public in interface? default implementation?

Reply:
fixed